src/components/devsupport/components/measure/measure.component.tsx

```diff
@@ -68,14 +68,14 @@ export const MeasureElement: React.FC<MeasureElementProps> = (props): MeasuringE
       measureSelf();
     } else {
       const originY = props.shouldUseTopInsets ? y + StatusBar.currentHeight || 0 : y;
-      const frame: Frame = bindToWindow(new Frame(x, originY, w, h), Frame.window());
+      const frame: Frame = bindToWindow(new Frame(x, originY, Math.round(w), Math.round(h)), Frame.window());
       props.onMeasure(frame);
     }
   };
 
   const measureSelf = (): void => {
     const node: number = findNodeHandle(ref.current);
-    UIManager.measureInWindow(node, onUIManagerMeasure);
+    if (typeof node === 'number') UIManager.measureInWindow(node, onUIManagerMeasure);
```

Comment:
Applying this breaks OverflowMenu(it stops displaying) on the web. `node` is object
```suggestion
    if (typeof node === 'number' || typeof node === 'object') UIManager.measureInWindow(node, onUIManagerMeasure);
```

Reply:
Actually, I've got the error using my suggestion, when the node is null on ios(so `typeof node === 'object'` is true). So maybe we could:
```
        if (!node) {
            return;
        }

        react_native_1.UIManager.measureInWindow(node, onUIManagerMeasure);
```